vault/plan.sh

```diff
@@ -1,17 +1,19 @@
 pkg_origin=core
 pkg_name=vault
-pkg_version=0.7.3
+pkg_version=0.9.1
 pkg_description="A tool for managing secrets."
 pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
 pkg_license=("MPL-2.0")
 pkg_upstream_url=https://www.vaultproject.io/
 pkg_source=https://releases.hashicorp.com/vault/${pkg_version}/vault_${pkg_version}_linux_amd64.zip
-pkg_shasum=2822164d5dd347debae8b3370f73f9564a037fc18e9adcabca5907201e5aab45
+pkg_shasum=6308013ee0d6278e98cdfe8d6de0162102a8d25f3bcd1e3737bf7b022a9f6702
 pkg_filename=${pkg_name}-${pkg_version}_linux_amd64.zip
 pkg_deps=()
 pkg_build_deps=(core/unzip)
 pkg_bin_dirs=(bin)
 pkg_svc_user=root
+pkg_svc_group=root
```

Comment:
What's the reason for removing this?

Reply:
Are we talking about the pkg_svc_group line?  I didn't remove it - i added it.  The package error-ed out, unable to build a directory until I added it.

Reply:
I can second this @elliott-davis. This was missing from the older core plan and when testing it, it fails during execution. This line needs to be here for the application to work.